lib/webPush.js

```diff
@@ -300,9 +300,9 @@ export const notifyReferral = async (userId) => {
   }
 }
 
-export const notifyInvite = async (userId) => {
+export const notifyInvite = async (userId, inviteId, inviteDescription) => {
   try {
-    await sendUserNotification(userId, { title: 'your invite has been redeemed', tag: 'INVITE' })
+    await sendUserNotification(userId, { title: `your invite ${inviteId ? `${inviteId} ` : ''}has been redeemed`, tag: 'INVITE' })
```

Comment:
I don't think this check makes sense, the id is always provided, it just might be random.

Reply:
right, i think i will just remove it as you've suggested, because showing `your invite difjdifuj093jfjkdifogjf has been redeemed` doesn't seem like good ux

Reply:
done